scene/main/viewport.cpp

```diff
@@ -162,6 +162,22 @@ ViewportTexture::~ViewportTexture() {
 }
 
 void Viewport::_sub_window_update_order() {
+	if (gui.sub_windows.is_empty())
+		return;
+
+	int index = gui.sub_windows.size() - 1;
+	if (!(gui.sub_windows[index].window->get_flag(Window::FLAG_ALWAYS_ON_TOP))) {
+		SubWindow sw = gui.sub_windows[index];
+
+		for (; index > 0; --index) {
+			if (!(gui.sub_windows[index - 1].window->get_flag(Window::FLAG_ALWAYS_ON_TOP)))
+				break;
+		}
+
+		gui.sub_windows.remove_at(gui.sub_windows.size() - 1);
+		gui.sub_windows.insert(index, sw);
```

Comment:
If `index == gui.sub_windows.size() - 1`, you remove the element and place it back to the same location.

Reply:
Added if statement